PeerioTechnologies/peerio-desktop
# chore: replace .dialog-warning classes with <Dialog theme=...>

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/8153/desktop-get-rid-of-old-dialog-warning-class

#### Testing instructions  
Before PeerUI we used the `dialog-warning` class on a few Dialog components to add the red/yellow line at the top of warning dialogs. PeerUI has this built in with the `theme` property (and the styling is slightly different). These dialogs have been changed and should now look consistent with the other warning dialogs in the app:

* signature error
* maintenance warning
* generic system warning
* update failed
* channel upgrade
  * (component still exists but is not currently being used)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
